link/uprobe.go

```diff
@@ -35,6 +35,19 @@ type Executable struct {
 	symbols map[string]elf.Symbol
 }
 
+// UprobeOptions defines additional parameters that will be used
+// when loading Uprobes.
+type UprobeOptions struct {
+	// Symbol offset. Must be provided in case of external symbols (shared libs).
+	// If set, overrides the offset eventually parsed from the executable.
+	Offset *uint64
+}
+
+// UprobeOffset is a manually provided offset.
+func UprobeOffset(offset uint64) *uint64 {
```

Comment:
I'd prefer to not have this function. It's very easy for users to write it themselves if they have to call this repeatedly. Otherwise the following is not too horrible imo:

```Go
offset := uint64(123)
Uprobe("main", prog, &UprobeOptions{Offset: &offset})
```

What do you think?

Reply:
👍🏻